src/assets/skins/yflink.scss

```diff
@@ -0,0 +1,9 @@
+.yflink {
```

Comment:
The skin need to be linked here otherwise it wont work: https://github.com/balancer-labs/snapshot/blob/5a3f8add163de337521316781e6173dced7ef1aa/src/assets/skins/index.scss